src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx

```diff
@@ -4,32 +4,87 @@ tags:
 
-<Callout variant="important">
-At a minimum, the `service.name` attribute must be present on your resource for data to be associated with an entity in New Relic.
+The following suites of attributes defined by the [OpenTelemetry resource semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#resource-semantic-conventions). These attributes are usually set by creating a resource using the OpenTelemetry SDK.
+
+  * `service.* attributes`
+    * `service.name` attribute is required to associate your resource with an entity in the UI
+    * `service.instance.id` is required for certain panes to light up
+  * `telemetry.sdk.language=java` is required to see data in the JVM section 
+
+## Batching [#batching]
+
+<Callout variant="caution">
+  Avoid getting rate limited! You should batch requests sent to the OTLP endpoint as described in this section.
 </Callout>
 
+By default, the OpenTelemetry SDKs and Collector send one (1) data point per request. Using these defaults, it is likely your account will be rate limited.
+
+All OpenTelemetry SDKs and Collectors provide a `BatchProcessor`, which batches data points in memory. This batching allows requests to be sent with more than one (1) data point.
+
+<table>
+  <thead>
+      <tr>
+        <th style={{ width: "200px" }}>Component</th>
+        <th>Batch Processor</th>
+      </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Collector</td>
+      <td>[Batch Processor](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/batchprocessor/README.md)</td>
+    </tr>
+    <tr>
+      <td>Go SDK</td>
+      <td>[BatchSpanProcessor](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#NewBatchSpanProcessor)</td>
+    </tr>
+    <tr>
+      <td>JS SDK</td>
+      <td>[BatchSpanProcessor](https://open-telemetry.github.io/opentelemetry-js/classes/_opentelemetry_tracing.batchspanprocessor.html)</td>
+    </tr>
+    <tr>
+      <td>Python SDK</td>
+      <td>[BatchExportSpanProcessor](https://open-telemetry.github.io/opentelemetry-python/sdk/trace.export.html#opentelemetry.sdk.trace.export.BatchExportSpanProcessor)</td>
+    </tr>
+  </tbody>
+</table>
+
 ## Traces [#traces]
 
-Make sure you are familiar with the options for sampling trace spans.
+Familiarize yourself with these trace topics to ensure your traces and spans appear in New Relic.
+
+### Start and end time [#start-end]
+
+The `startTimeUnixNano` and `endTimeUnixNano` fields are required according to the OpenTelemetry specification for trace data. When `startTimeUnixNano` is not present the span will be dropped. When `endTimeUnixNano` is not present the duration of your span will be large and negative.
+
+### IDs for traces and spans [#ids]
+
+The `traceId` and `spanId` fields are required according to the OpenTelemetry specification for trace data. When `traceId` or `spanId` are not present the span will be dropped.
```

Comment:
@mmfred Sounds like you're suggesting we drop the word "field." Not sure at this point.

Reply:
No, I was doing a subject verb agreement check. As is, the sentence should have an "is" instead of an "are" - so it would be: When `traceId` or `spanId` is not present, the span will be dropped.